app/controllers/responsiblepeople/PersonNonUKPassportController.scala

```diff
@@ -18,32 +18,29 @@ package controllers.responsiblepeople
 
 import config.AppConfig
 import connectors.DataCacheConnector
-import controllers.BaseController
+import controllers.DefaultBaseController
 import forms.{EmptyForm, Form2, InvalidForm, ValidForm}
 import javax.inject.Inject
 import models.responsiblepeople.{NonUKPassport, ResponsiblePerson}
 import play.api.i18n.MessagesApi
 import play.api.mvc.{AnyContent, Request}
 import uk.gov.hmrc.http.cache.client.CacheMap
-import uk.gov.hmrc.play.frontend.auth.AuthContext
-import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
-import utils.{ControllerHelper, RepeatingSection}
+import utils.{AuthAction, ControllerHelper, RepeatingSection}
 import views.html.responsiblepeople.person_non_uk_passport
 
 import scala.concurrent.Future
 
 class PersonNonUKPassportController @Inject()(
                                             override val messagesApi: MessagesApi,
                                             val dataCacheConnector: DataCacheConnector,
-                                            val authConnector: AuthConnector,
+                                            authAction: AuthAction,
                                             val appConfig:AppConfig
-                                          ) extends RepeatingSection with BaseController {
+                                          ) extends RepeatingSection with DefaultBaseController {
```

Comment:
if you could align here would be nice :)

Reply:
Done